Frontend/src/services/request.js

```diff
@@ -1,17 +1,17 @@
 
 import axios from 'axios';
-import config from "../const.js";
+import dev_const from "../const.js";
 
 
-let apiURL = localStorage.getItem('base') || config.API_ROUTE;
+let apiURL = localStorage.getItem('base') || `${window.location.protocol}//${window.location.hostname}:${dev_const.DEV_PORT}/api`;
 
 /* eslint-disable */
 let token = localStorage.getItem('token');
 
 const register = registrationData => {
   return axios.post(`${apiURL}/users/register`, registrationData)
     .then(response => {
-      token = response.data.token;
+      token = response.data.data.token;
```

Comment:
None of these changes should be present. These are all reversions of the changes I made in my branch.